### PR TITLE
fix: do not check for browser extension in mobile app

### LINKED ISSUE #57469: freeCodeCamp iOS app bug - 'You appear to be using a browser extension that is modifying the page'
title

Image

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #57469

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md [hunk 1]
```diff
@@ -49,9 +49,13 @@ assert(document.querySelector('h1').innerText.toLowerCase() === 'catphotoapp');
...
-assert.isAtMost(document.querySelectorAll('script').length, 2);
-assert.equal(document.querySelectorAll('style').length, 1);
-assert.equal(document.querySelectorAll('link').length, 0);
+if(__checkForBrowserExtensions){
+  assert.isAtMost(document.querySelectorAll('script').length, 2);
+  assert.equal(document.querySelectorAll('style').length, 1);
+  assert.equal(document.querySelectorAll('link').length, 0);
+} else {
+  assert(true);
+}
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md:58]:
```suggestion
}
```
I don't think `assert(true)` does anything.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
There's also curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/5dc174fcf86c76b9248c6eb2.md, i.e. the FSD version of this file.

**Comment by Sembauke** [comment]
Well in the mobile app we handle testing challenges differently so the number of script/style tags are different.

**Comment by a2937** [comment]
> Also, why does the mobile app have this issue? Thing is, this feels like it shouldn't be necessary, even though I can see how it can work.

My theory (that I can't prove yet) was that there was no inherited stylesheet in the mobile app. The mobile app is using a DOM parser and I don't see it inventing a new stylesheet.

**Review by ojeytonwilliams — APPROVED** [review]
Ideally, the mobile and Web apps should test the same way, but until we address that, I think this will do.

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**